django/db/backends/postgresql/base.py

```diff
@@ -1,13 +1,13 @@
 """
 PostgreSQL database backend for Django.
 
-Requires psycopg 2: https://www.psycopg.org/
+Requires psycopg2 >= 2.8.4 or psycopg3
```

Comment:
With the usage of ClientCursor, I believe at least psycopg 3.1 is required.

Reply:
ACK, will update that as well.